CMakeLists.txt

```diff
@@ -425,7 +425,7 @@ endif ()
 if (HAVE_SETJMP_H)
   set(WASM_RT_FILES "wasm2c/wasm-rt-impl.h" "wasm2c/wasm-rt-impl.c" "wasm2c/wasm-rt-exceptions-impl.c")
 
-  add_library(wasm-rt-impl STATIC ${WASM_RT_FILES})
+  add_library(wasm-rt-impl ${WASM_RT_FILES})
```

Comment:
I don't think this particular library ever wants to be shared.   I could perhaps be wrong.